packages/database/src/database/tokenBoundAccounts.ts

```
@@ -0,0 +1,49 @@
+  let skip = 0
+  let take = 100
+  let orderBy = {}
+  let include = {}
+  let filter = { where, skip, take, orderBy }
+  if (query?.page || query?.size) {
+    let page = parseInt(query?.page || '0')
+    let size = parseInt(query?.size || '100')
+    if (page < 0) { page = 0 }
+    if (size < 0) { size = 100 }
+    if (size > 200) { size = 200 }
+    let start = page * size
```

Biome

[error] This let declares a variable that is only assigned once.

'start' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)